src/components/action-popover/action-popover.spec.js

```diff
@@ -207,6 +214,41 @@ describe("ActionPopover", () => {
     render(null, DOM);
   });
 
+  describe("if download prop and href prop are provided", () => {
+    it("should render as a link component", () => {
+      const clickFn = jest.fn();
+      wrapper = enzymeMount(
+        <ThemeProvider theme={mintTheme}>
+          <ActionPopover>
+            <ActionPopoverItem key="1" onClick={clickFn} href="#" download>
+              test download
+            </ActionPopoverItem>
+          </ActionPopover>
+        </ThemeProvider>
+      );
+
+      act(() => {
+        wrapper
+          .find(MenuButton)
+          .props()
+          .onClick({ stopPropagation: () => {} });
+      });
+
+      act(() => {
+        wrapper.update();
+      });
+
+      act(() => {
+        wrapper
+          .find(StyledDiv)
+          .props()
+          .onKeyDown({ which: 13, stopPropagation: jest.fn() });
+      });
```

Comment:
These tests should be split into 2 - you should test that clicking works in one, then that pressing enter works in another. 
This test would pass if either clicking or pressing enter worked, one of them could not work and it would still pass

Reply:
This was me misreading the code - no action required